transformers4rec/torch/model/prediction_task.py

```diff
@@ -44,6 +44,82 @@ def build(self, input_size) -> SequentialBlock:
 
 
 class BinaryClassificationTask(PredictionTask):
+    """Returns a ``PredictionTask`` for binary classification.
+
+    Example usage::
+
+        # Define the input module to process the tabular input features.
+        input_module = tr.TabularSequenceFeatures.from_schema(
+            schema,
+            max_sequence_length=max_sequence_length,
+            continuous_projection=d_model,
+            aggregation="concat",
+            masking=None,
+        )
+
+        # Define XLNetConfig class and set default parameters for HF XLNet config.
+        transformer_config = tr.XLNetConfig.build(
+            d_model=d_model, n_head=4, n_layer=2, total_seq_length=max_sequence_length
+        )
+
+        # Define the model block including: inputs, masking, projection and transformer block.
+        body = tr.SequentialBlock(
+            input_module,
+            tr.MLPBlock([64]),
+            tr.TransformerBlock(
+                transformer_config,
+                masking=input_module.masking
+            )
+        )
+
+        # Define a head with BinaryClassificationTask.
+        head = tr.Head(
+            body,
+            tr.BinaryClassificationTask(
+                "click",
+                summary_type="mean",
+                metrics=[
+                    tm.Precision(task='binary'),
+                    tm.Recall(task='binary'),
+                    tm.Accuracy(task='binary'),
+                    tm.F1Score(task='binary')
+                ]
+            ),
+            inputs=input_module,
+        )
```

Comment:
It looks good - can we add the line until, we have the `model` object?

Reply:
thanks.

Reply:
Added up to the model object.